src/main/java/net/apnic/rdap/autnum/filters/AutnumRouteFilter.java

```diff
@@ -0,0 +1,50 @@
+package net.apnic.rdap.autnum.filters;
+
+import net.apnic.rdap.authority.RDAPAuthority;
+import net.apnic.rdap.autnum.AsnRange;
+import net.apnic.rdap.directory.Directory;
+import net.apnic.rdap.error.MalformedRequestException;
+import net.apnic.rdap.filter.filters.RDAPPathRouteFilter;
+import net.apnic.rdap.filter.RDAPRequestPath;
+import net.apnic.rdap.filter.RDAPRequestType;
+import net.apnic.rdap.resource.ResourceNotFoundException;
+
+/**
+ *
+ */
+public class AutnumRouteFilter
```

Comment:
Don't know if I agree. The code that still has to be written will make great use of heavy lifting done in the parent class. Trying to make it so that these child classes only have worry about finding the RDAPAuthority and the parent takes it from there.

From what I can see any other solution would risk doubling up logic?